emcc.py

```diff
@@ -2771,14 +2771,19 @@ def modularize():
       # after document.currentScript is gone, so we save it.
       # (when MODULARIZE_INSTANCE, an instance is created
       # immediately anyhow, like in non-modularize mode)
+      if shared.Settings.EXPORT_ES6:
+        GETSCRIPT = "import.meta.url"
+      else:
+        GETSCRIPT = "typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined"
       src = '''
 var %(EXPORT_NAME)s = (function() {
-  var _scriptDir = typeof document !== 'undefined' && document.currentScript ? document.currentScript.src : undefined;
+  var _scriptDir = %(GETSCRIPT)s;
```

Comment:
No need to uppercase here either.  EXPORT_NAME is uppercase because it corresponds to a Settings member.

Reply:
Fair enough. I'll change this.